pkg/apis/minio.min.io/v2/types.go

```diff
@@ -609,18 +611,19 @@ type CustomCertificateConfig struct {
 //
 // See the https://min.io/docs/minio/kubernetes/upstream/operations/install-deploy-manage/deploy-minio-tenant.html#procedure-command-line[MinIO Operator CRD] reference for the `pools` object for examples and more complete documentation. +
 type Pool struct {
-	// *Optional* +
+	// *Required*
```

Comment:
This may break legacy pools without a name, we should document this change

Reply:
Not may. It will break existing pools. I don't get why this change was approved and even released already (btw. as a bug fix release - do you follow SemVer or not) without any existing documentation. Sorry to say, but the entire release process for the operator is somehow broken.